### DISCUSSION TITLE
Where is "dump/loftr_ds_indoor/LoFTR_pred_eval.npy" in visualize_dump_results in notebook?

### DISCUSSION THREAD

**Discussion by HUSTNO1WXY** [zju3dv/LoFTR#63] [Q&A] [answered]
I notice there is 
npy_path = "dump/loftr_ds_indoor/LoFTR_pred_eval.npy"
in visualize_dump_results which is in notebook. But I can't find it. 
Could you give me some help? Thanks.

**Comment by zehongs** [comment, marked as answer]
hi, we do not provide it. You need to generate it by yourself.
